mm2src/mm2_test_helpers/src/for_tests.rs

```diff
@@ -159,18 +159,25 @@ pub const MARTY_ELECTRUM_ADDRS: &[&str] = &[
 ];
 pub const ZOMBIE_TICKER: &str = "ZOMBIE";
 pub const ARRR: &str = "ARRR";
+#[cfg(not(target_arch = "wasm32"))]
 pub const ZOMBIE_ELECTRUMS: &[&str] = &[
     "electrum1.cipig.net:10008",
     "electrum2.cipig.net:10008",
     "electrum3.cipig.net:10008",
 ];
+#[cfg(target_arch = "wasm32")]
+pub const ZOMBIE_ELECTRUMS: &[&str] = &["zombie.dragonhound.info:30059"];
+
+#[cfg(not(target_arch = "wasm32"))]
 pub const ZOMBIE_LIGHTWALLETD_URLS: &[&str] = &[
     "https://lightd1.pirate.black:443",
     "https://piratelightd1.cryptoforge.cc:443",
     "https://piratelightd2.cryptoforge.cc:443",
     "https://piratelightd3.cryptoforge.cc:443",
     "https://piratelightd4.cryptoforge.cc:443",
 ];
+#[cfg(target_arch = "wasm32")]
+pub const ZOMBIE_LIGHTWALLETD_URLS: &[&str] = &["http://pirate.spyglass.quest:8081"];
```

Comment:
Please rename this const as its semantics don't match the name.

Reply:
I don't get you very clear...more urls could be added and the current url is only for testing purpose.

Reply:
The URL contains `pirate` and ticker used in tests is `ARRR`, which is confusing:
https://github.com/KomodoPlatform/komodo-defi-framework/blob/fab33076f4d9daf17314fd0fda6d4b8a64ba5b97/mm2src/mm2_main/src/wasm_tests.rs#L234

If this is used for PIRATE, then it should be named `PIRATE_LIGHTWALLETD_URLS`.

Reply:
okay thanks.